.env

```
@@ -1 +1,2 @@
-REACT_APP_API_URL=http://localhost:3001/api
+GUARANI_API_URL='http://cidia.unahur.edu.ar/api-guarani'
+REACT_APP_API_URL='http://localhost:3001/api'
```

El front no le pega a Guaraní, y la URL de la API estaba bien puesta.

```suggestion
REACT_APP_API_URL=http://localhost:3001/api
```